backend/dal/Services/Concrete/ParcelService.cs

```diff
@@ -192,6 +196,17 @@ public Parcel Add(Parcel parcel)
             parcel.Classification = this.Context.PropertyClassifications.Find(parcel.ClassificationId);
             parcel.IsVisibleToOtherAgencies = false;
 
+            parcel.Parcels.ForEach(pp =>
```

Comment:
My brain finds the two `ForEach(...)` statements hard to read.

Reply:
I'll add some comments, and split this logic up with an if statement since there should never be a time that both for loops run for the same parcel.